card/mod/pointer/spec/set/abstract/pointer/html_views_spec.rb

```diff
@@ -0,0 +1,113 @@
+# -*- encoding : utf-8 -*-
+
+describe Card::Set::Abstract::Pointer do
+  extend Card::SpecHelper::ViewHelper::ViewDescriber
+
+  context "with simple Pointer" do
+    let(:pointer) do
+      pointer = Card["Sample Pointer"]
+      pointer << "A"
+      pointer
+    end
+
+    describe_views :core, :closed_content, :editor,
+                   :list, :autocomplete, :checkbox,
+                   :radio, :select, :multiselect do |view|
+      it "doesn't have errors" do
```

Comment:
these 9 tests are new (and at least one – autocomplete – would have failed before)